package.json

```diff
@@ -1,7 +1,14 @@
 {
   "name": "sidechain",
   "esy": {
-    "build": "dune build -p sidechain"
+    "build": "dune build -p #{self.name}",
+    "release": {
+      "rewritePrefix": true,
```

Comment:
https://esy.sh/docs/en/npm-release.html was recommended here, not totally sure of the pros/cons

Reply:
I would like if we avoided it, because then we don't rely on it, and if we don't rely on it it's not needed. The goal is to be able to ship sidechain static binaries for linux without any work.

Reply:
Ok, I removed it, it works fine without it anyway